README.md

```diff
@@ -200,7 +265,7 @@ Equity compensation awards can give rise to federal and state income taxes as we
-- [Alternative Minimum Tax (AMT)](http://fairmark.com/general-taxation/alternative-minimum-tax/alternative-minimum-tax-101/) is a [complex part](https://www.irs.gov/taxtopics/tc556.html) of the federal tax code many taxpayers never worry about. Generally, you do not pay [unless you have high income (>$250K) or high deductions](http://www.marketwatch.com/story/congratulations-you-owe-the-alternative-minimum-tax-2014-01-14). It also depends on the state you’re in, since your state taxes can significantly affect your deductions. Confusingly, if you are affected, AMT tax rates are usually at **26%** or **28%** marginal tax rate, but effectively is **35%** for some ranges, meaning it is [higher than ordinary income tax for some incomes and lower for others](http://www.forbes.com/sites/feeonlyplanner/2011/12/16/the-alternative-minimum-tax-sweet-spot/). AMT rules are so complicated you often need professional tax help if they might apply to you. The IRS’s [AMT Assistant](https://www.irs.gov/Businesses/Small-Businesses-&-Self-Employed/Alternative-Minimum-Tax-(AMT)-Assistant-for-Individuals) might also help.
+- [Alternative Minimum Tax (AMT)](http://fairmark.com/general-taxation/alternative-minimum-tax/alternative-minimum-tax-101/) is a [complex part](https://www.irs.gov/taxtopics/tc556.html) of the federal tax code many taxpayers never worry about. Generally, you do not pay [unless you have high income (>$250K) or high deductions](http://www.marketwatch.com/story/congratulations-you-owe-the-alternative-minimum-tax-2014-01-14). It also depends on where you file, since your state taxes can significantly affect your deductions. Confusingly, if you are affected, AMT tax rates are usually at **26%** or **28%** marginal tax rate, but effectively is **35%** for some ranges, meaning it is [higher than ordinary income tax for some incomes and lower for others](http://www.forbes.com/sites/feeonlyplanner/2011/12/16/the-alternative-minimum-tax-sweet-spot/). AMT rules are so complicated you often need professional tax help if they might apply to you. The IRS’s [AMT Assistant](https://www.irs.gov/Businesses/Small-Businesses-&-Self-Employed/Alternative-Minimum-Tax-(AMT)-Assistant-for-Individuals) might also help.
 - ❗ AMT is important to understand because exercising incentive stock options can trigger AMT. In some cases a *lot* of AMT, *even when you haven’t sold the stock* and have no money to pay. More on this below.
 - 🔹 Section 1202 of the Internal Revenue Code provides a special tax break for qualified small business stock held for more than five years. Currently, this tax break is a 100% exclusion from income for up to $10M in gain. There are also special rules that enable you to rollover gain on qualified small business stock you have held for less than five years. Stock received on the exercise of options can qualify for the Section 1202 stock benefit.
```

Comment:
Qualified small business stock? What's that? Is that any stock in a small business? How small? When does exercising qualify for this benefit? When doesn't it?

Reply:
🚧